src/SDK/Trace/SpanBuilder.php

```diff
@@ -51,17 +53,9 @@ public function __construct(
     }
 
     /** @inheritDoc */
-    public function setParent(Context $parentContext): API\SpanBuilderInterface
+    public function setParent($context): API\SpanBuilderInterface
```

Comment:
I think we should document here (or in the API?) what ContextInterface/null/false mean

Reply:
`SpanBuilderInterface::setParent()`:
```
 * @param ContextInterface|false|null $context the parent context, null to use the
 *        current context, false to set no parent
```